### ISSUE TITLE
Dearest Natalia (8304) (BB #909)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#909]
This issue was migrated from bitbucket.
**Original Reporter:** bewtihozzy
**Original Date:** 26.05.2015 18:30:29 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/909

<hr>

Commander Mar'alith at Cenarion Hold in Silithus wants you to question the inhabitants of Bronzebeard's Encampment. 

Frankal questioned
Rutgar questioned

Everything looks good with Rutgar but when u question Frankal it doesnt count for the quest.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 28.05.2015 19:42:37 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/909/#comment-18437452

<hr>

Fix #898, #912 & #909

→ <<cset 662f81fc79f4>>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 30.05.2015 01:22:31 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/909/#comment-18473902

<hr>

[Core/Opcodes] Fix a sanity check in HandleGameObjectUseOpcode that was added to
               block certain exploits. As far as I'm aware, the fix is still valid
               but now we have returned a majority of broken gameobjects to normal.

```
           Fixes #931, #943.
           Correctly fixes #912, #909 and #898.
```

→ <<cset e215d2efab6f>>
